### PR TITLE
Formatting and screenshots

### PR DESCRIPTION
**Pull request by aaronp**
Hi @paulpbrandon  - really great work! Keen to go through this with you - I hope I've not stomped too much on your great work!

### CODE REVIEW

File: installcerts.sh [hunk 1]
```diff
@@ -1,6 +1,8 @@
 #!/bin/sh
 
-/Library/Frameworks/Python.framework/Versions/3.10/bin/python3.10 << "EOF"
+# you may or may not need to change the location of Python on line 3 to the one the CLI is using.
```
Comment (paulpbrandon) [inline installcerts.sh]: It's not line 3 anymore now

File: docs/azure-aks.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
+[back](../readme.md)
```
Comment (paulpbrandon) [inline docs/azure-aks.md]: Back link is wrong, should be uppercase,

File: README.md [hunk 3]
```diff
@@ -1,106 +1,138 @@
...
+See [here](./docs/installK8s.md)
+
+From here, we assume `kubectl` is installed/authenticated.
+
+### 2) Deploy Argo to Kubernetes
+
+These steps assume the following environment variables:
+```
+export APP_NAME=example
+export ARGO_NAMESPACE="${APP_NAME}-argocd"
```
Comment (paulpbrandon) [inline README.md]: If not configuring for the namespace "argocd" we will need to add a note, as there will be more work

As per the Argo docs:
_Warning

The installation manifests include ClusterRoleBinding resources that reference argocd namespace. If you are installing Argo CD into a different namespace then make sure to update the namespace reference._

Or we just altering this readme to go with using argocd as the namespace

File: installcerts.sh (hunk 1 above)
Comment (aaronp) [reply installcerts.sh]: Cheers Paul - fixed.

**Pull request closed without merging**